index.html

```diff
@@ -0,0 +1,290 @@
+                <div class="card-image">
+                  <img src="./images/img1.jpg" alt="card-work">
+                </div>
+                <div class="left1-block">
+                  <h2 class="h2">
+                    Multi-Post
+                    Stories
+                  </h2>
+                  <div class="frame2">
+                    <p>CONOPY</p>
+                      <img src="./images/Counter.png" alt="">
+                    <p class="p1">Back End Dev</p>
+                      <img src="./images/Counter.png" alt="">
+                    <p class="p1">2020</p>
+                  </div>
+                  <p class="primary2">
+                    A daily selection of privately<br />personalized reads; 
+                    no accounts or<br /> sign-ups required.
+                  </p>
+                  <ul>
+                    <li><a href="#">html</a></li>
+                    <li><a href="#">css</a></li>
+                    <li><a href="#">javascript</a></li>
+                  </ul>
+          <div class="action1">
+            <p>See Project</p>
+          </div>
+        </div>
+      </div>
+      <div class="card-work">
+        <div class="card-image">
+          <img src="./images/3.png" alt="card-work">
+        </div>
+        <h2 class="h2">Tonic</h2>
+      <div class="left2-block">
+          <div class="frame3">
+            <p>CONOPY</p>
+            <img src="./images/Counter.jpg" alt="">
+            <p class="p1">Back End Dev</p>
+            <img src="./images/Counter.jpg" alt="">
+            <p class="p1">2020</p>
+          </div>
+          
+            <p class="primary3">
+              A daily selection of privately<br />personalized reads; 
+            no accounts or<br /> sign-ups required.
+          </p>
+          <ul>
+            <li><a href="#">html</a></li>
+            <li><a href="#">css</a></li>
+            <li><a href="#">javascript</a></li>
+          </ul>
+          <div class="action2">
+            <p>See Project</p>
+          </div>
+          </div>
+      </div>  
+      <div class="card-work">
+        <div class="card-image">
+          <img src="./images/4.png" alt="card-work">
+          <h2 class="h2">
+            Multi-Post
+            Stories
+          </h2>
+        </div> 
+        <div class="left4-block">
+            <div class="frame4">
+              <p>CONOPY</p>
+              <img src="./images/Counter.jpg" alt="">
+              <p class="p1">Back End Dev</p>
+              <img src="./images/Counter.jpg" alt="">
+              <p class="p1">2020</p>
+            </div>
+        <p class="primary4">
+          A daily selection of privately<br />personalized reads; 
+          no accounts or<br /> sign-ups required.
+      </p>
+     <ul>
+        <li><a href="#">html</a></li>
+        <li><a href="#">css</a></li>
+        <li><a href="#">javascript</a></li>
+      </ul>
+      <div class="action3">
+        <p>See Project</p>
+      </div>
```

Comment:
- [ ] The button ` See project` should be a link or button tag, and follow the button styling: Active, Hover, and Pressed (see design). Kindly implement it
### Figma button design
![figma](https://user-images.githubusercontent.com/81727988/165932664-35948504-c2b3-4372-b10e-47c7cadc81e2.png)